### ISSUE TITLE
IOS 13.0 - Hybrid app - Webkit Response timeout - Received WebKit data: '{"error":{"code":-32000,"message"

### ISSUE THREAD

**Issue by githubdhina** [appium/appium#13386]
## The problem
Webkit response times out with error
Received WebKit data: '{"error":{"code":-32000,"message":"Target not found.","data":[{"code":-32000,"message":"Target not found."}]},"id":1}'
[RemoteDebugger] Received receipt for message '1'
[XCUITest] TimeoutError: operation timed out
[XCUITest]     at afterTimeout (/Applications/Appium_1.15.app/Contents/Resources/app/node_modules/appium/node_modules/bluebird/js/release/timers.js:46:19)
[XCUITest]     at Timeout.timeoutTimeout [as _onTimeout] (/Applications/Appium_1.15.app/Contents/Resources/app/node_modules/appium/node_modules/bluebird/js/release/timers.js:76:13)
[XCUITest]     at ontimeout (timers.js:427:11)
[XCUITest]     at tryOnTimeout (timers.js:289:5)
[XCUITest]     at listOnTimeout (timers.js:252:5)
[XCUITest]     at Timer.processTimers (timers.js:212:10)
[WD Proxy] Matched '/session/10f6f8e9-7d2c-4738-a7fd-7d6cd3da3307' to command name 'deleteSession'

## Environment

* Appium version - 1.15.0
* Last Appium version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Appium: 1.15.0
* Mobile platform/version under test: IOS  version 13.0, Hybrid App
* Real device or emulator/simulator: Real Device
* IOS Web Kit Proxy: Latest from Head

## Details
Unable to switch to WebView successfully with the appium capabilities below.

## Link to Appium logs

https://gist.github.com/githubdhina/4d7cb8c8dc8976332d40a53f8e40851c

## Code To Reproduce Issue [ Good To Have ]

[BaseDriver] Event 'newSessionRequested' logged at 1570530644037 (18:30:44 GMT+0800 (+08))
[Appium] Appium v1.13.0 creating new XCUITestDriver (v2.113.2) session
[Appium] Capabilities:
[Appium]   autoWebview: true
[Appium]   automationName: XCUITest
[Appium]   bundleId: com.XXXXX_com.XXXXXX
[Appium]   clearSystemFiles: true
[Appium]   deviceName: Test iPhone
[Appium]   ignoreUnimportantViews: true
[Appium]   language: en
[Appium]   locale: US
[Appium]   maxTypingFrequency: 120
[Appium]   newCommandTimeout: 300
[Appium]   noReset: true
[Appium]   platformName: iOS
[Appium]   platformVersion: 13.0
[Appium]   startIWDP: true
[Appium]   udid: XXXXX
[Appium]   webkitResponseTimeout: 5000
[Appium]   connectHardwareKeyboard: true

**Comment by mykola-mokhnach** [comment]
Please check if this works with appium@beta

**Comment by githubdhina** [comment]
Works with appium@beta. Followed steps on #13285 to fix this.

What I did:

clone appium if you have not already (git clone https://github.com/appium/appium.git)
go into the appium source (cd appium)
change branches (git checkout isaac-target-poll)
install the dependencies (npm install)
Copy the entire node_modules dir resulted from the build, in the Appium 1.15 installation path /usr/local/lib/node_modules/appium/node_modules; basically I replaced the directory node_modules here.
I remade the appium-webdriveragent manual configuration (this is how we use it on our project)
I ran the automated tests which perform javascript execution and no hanging happened.
The Appium server & driver are started from the Java code.
